source/nodejs/adaptivecards/src/serialization.ts

```diff
@@ -840,7 +865,7 @@ export type PropertyBag = { [propertyName: string]: any };
 
 export abstract class SerializableObject {
     static onRegisterCustomProperties?: (sender: SerializableObject, schema: SerializableObjectSchema) => void;
-    static defaultMaxVersion: Version = Versions.latest;
+    static defaultMaxVersion: Version = Versions.maxSupportedVersion;
```

Comment:
Why is this new `maxSupportedVersion` even necessary? `Version.latest` is meant to mean exactly that: it's the latest (max) supported version. The two are now redundant unless I am missing something.

Reply:
add declaredVersions which can be shared by both designer and serialization